### PR TITLE
Fixed unsafe weight handling

### PR DESCRIPTION
**Pull request by diogovalada**
Instead of storing the NN's weights before the Learning rate search in a "tmp.h5", they are not just kept in memory.
This way, different instances will not interfere.

### CODE REVIEW

File: keras_lr_finder/lr_finder.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def find(self, x_train, y_train, start_lr, end_lr, batch_size=64, epochs=1):
         num_batches = epochs * N / batch_size
         self.lr_mult = (float(end_lr) / float(start_lr)) ** (float(1) / float(num_batches))
         # Save weights into a file
-        self.model.save_weights('tmp.h5')
+        self.weights = self.model.get_weights()
```
Comment (surmenok) [inline keras_lr_finder/lr_finder.py]: It doesn't have to be a class attribute. Can we just make it a local variable?

Comment (diogovalada) [reply keras_lr_finder/lr_finder.py]: Yeah, of course. I can arrange that.

Comment (diogovalada) [reply keras_lr_finder/lr_finder.py]: It is now an in-function variable called initial_weights, as of the last commit.
